<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetMMainWindow</class>
 <widget class="QMainWindow" name="AssetMMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AssetMMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>741</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(197, 197, 197);</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_Site">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>139</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Site</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>201</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpload">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>290</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 51, 154);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Site">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>350</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Enter Site</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>171</width>
       <height>280</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_description">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_srNo">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter SrNo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Manufacture">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter Manufacture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Model">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Owner">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="placeholderText">
         <string>Enter Owner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_condition">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>Choose Condition</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Good</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bad</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_location">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>Choose Location</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pune</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hydrabad</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>141</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Type">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Description">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_SrNo">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>SrNo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Manufacture">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Manufacture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Model">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Owner">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Owner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Condition">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Condition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Loacation">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Loacation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 51, 154);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>500</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 51, 154);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainWindowClose">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
